src/main/js/PipelineEditorLink.jsx

```diff
@@ -15,7 +15,9 @@ class PipelineEditorLink extends React.Component {
         const href = Paths.rest.apiRoot() + '/organizations/' + pipeline.organization + '/pipelines/' + folder + '/';
         pipelineService.fetchPipeline(href, { useCache: true })
         .then(pipeline => {
-            if (pipeline._class === 'io.jenkins.blueocean.blueocean_github_pipeline.GithubOrganizationFolder') {
+            if (pipeline._class === 'io.jenkins.blueocean.blueocean_github_pipeline.GithubOrganizationFolder' ||
```

Comment:
No, no requirement for backwards compatibility, the editor is included as a BO dependency, so BO specifies the version.